pokemongo_bot/handlers/catch_handler.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from cell_workers import PokemonCatchWorker
+from cell_workers.utils import distance
+from events import IEventHandler
+
+class CachablePokemonHandler(IEventHandler):
```

Comment:
ty - I was thinking of cache.  Note, this is still heavily WIP - I just wanted to add some progress